templates/focus-ios.xliff

```diff
@@ -181,7 +179,6 @@
       </trans-unit>
       <trans-unit id="Autocomplete.defaultLabel">
         <source>Autocomplete</source>
-        <note>Label for enabling or disabling autocomplete</note>
```

Comment:
Is there a reason we are dropping three localization notes from the strings above?

Reply:
This is what happens when we remove strings from UIConstants. I thought it had been fixed. I'll submit a new PR that keeps these redundant strings around.